dev/com.ibm.ws.request.probe.audit.servlet/src/com/ibm/ws/request/probe/audit/servlet/AuditPE.java

```diff
@@ -728,8 +728,12 @@ private void auditEventSafAuth(Object[] methodParams) {
         String applid = (String) varargs[8];
         String accessLevel = (String) varargs[9];
         String errorMessage = (String) varargs[10];
+		String methodName = (String) varargs[11];
```

Comment:
Don't you need to handle the case where the 11th arg isn't there?  If someone is running the current level of WS-CD-Open with this version of Open Liberty, won't we take an IndexOutOfBoundsException?